tests/testthat/test-score_model_out_rel_metrics.R

```diff
@@ -0,0 +1,294 @@
+    ))) |>
+    dplyr::summarize(
+      ae_median = sum(ifelse(
+        .data[["output_type_id"]] == 0.5,
+        abs(.data[["oracle_value"]] - .data[["value"]]),
+        0
+      )),
+      wis = 2 * mean(.data[["qs"]]),
+      interval_coverage_80 = (sum(.data[["q_coverage_80_lower"]], na.rm = TRUE) == 1) *
+        (sum(.data[["q_coverage_80_upper"]], na.rm = TRUE) == 1),
+      interval_coverage_90 = (sum(.data[["q_coverage_90_lower"]], na.rm = TRUE) == 1) *
+        (sum(.data[["q_coverage_90_upper"]], na.rm = TRUE) == 1)
+    )
+
+  exp_scores_standard <- exp_scores_unsummarized |>
+    dplyr::group_by(dplyr::across(dplyr::all_of(
+      c("model_id", "location")
+    ))) |>
+    dplyr::summarize(
+      ae_median = mean(.data[["ae_median"]]),
+      wis = mean(.data[["wis"]]),
+      interval_coverage_80 = mean(.data[["interval_coverage_80"]], na.rm = TRUE),
+      interval_coverage_90 = mean(.data[["interval_coverage_90"]], na.rm = TRUE),
+      .groups = "drop"
+    )
+
+  exp_scores_relative_ae_median <- get_pairwise_scores_by_loc(exp_scores_unsummarized, "ae_median")
+  exp_scores_relative_wis <- get_pairwise_scores_by_loc(exp_scores_unsummarized, "wis")
+  exp_scores <- exp_scores_standard |>
+    dplyr::full_join(exp_scores_relative_ae_median, by = c("model_id", "location")) |>
+    dplyr::full_join(exp_scores_relative_wis, by = c("model_id", "location"))
+
+  # same column names, number of rows, and score values
+  expect_equal(colnames(act_scores), colnames(exp_scores))
+  expect_equal(nrow(act_scores), nrow(exp_scores))
+  merged_scores <- dplyr::full_join(
+    act_scores, exp_scores,
+    by = c("model_id", "location")
+  )
+  expect_equal(nrow(act_scores), nrow(merged_scores))
+  expect_equal(merged_scores$ae_median.x, merged_scores$ae_median.y)
+  expect_equal(merged_scores$wis.x, merged_scores$wis.y)
+  expect_equal(merged_scores$interval_coverage_80.x, merged_scores$interval_coverage_80.y)
+  expect_equal(merged_scores$interval_coverage_90.x, merged_scores$interval_coverage_90.y)
+  expect_equal(merged_scores$ae_median_relative_skill.x, merged_scores$ae_median_relative_skill.y)
+  expect_equal(merged_scores$wis_relative_skill.x, merged_scores$wis_relative_skill.y)
+})
+
+
+test_that("score_model_out succeeds with valid inputs: quantile output_type, relative wis and ae, Flusight-baseline", {
+  # Forecast data from hubExamples: <https://hubverse-org.github.io/hubExamples/reference/forecast_data.html>
+  forecast_outputs <- hubExamples::forecast_outputs
+  forecast_oracle_output <- hubExamples::forecast_oracle_output
+
+  act_scores <- score_model_out(
+    model_out_tbl = forecast_outputs |> dplyr::filter(.data[["output_type"]] == "quantile"),
+    oracle_output = forecast_oracle_output,
+    metrics = c("ae_median", "wis", "interval_coverage_80", "interval_coverage_90"),
+    relative_metrics = c("ae_median", "wis"),
+    baseline = "Flusight-baseline",
+    by = c("model_id", "location")
+  )
```

Comment:
There are four warnings from wilcox.test. This captures all of them so you don't clutter the test output with warnings. 

```suggestion
  (
    act_scores <- score_model_out(
      model_out_tbl = forecast_outputs |> dplyr::filter(.data[["output_type"]] == "quantile"),
      oracle_output = forecast_oracle_output,
      metrics = c("ae_median", "wis", "interval_coverage_80", "interval_coverage_90"),
      relative_metrics = c("ae_median", "wis"),
      baseline = "Flusight-baseline",
      by = c("model_id", "location")
    )
  )%>% 
    expect_warning("cannot compute exact p-value") %>%
    expect_warning("cannot compute exact p-value") %>%
    expect_warning("cannot compute exact p-value") %>%
    expect_warning("cannot compute exact p-value")
```